tests/docker_run_tests.rs

```diff
@@ -163,6 +163,13 @@ fn test_pnpm_custom_version() {
     assert!(output.contains("Hello from PNPM"));
 }
 
+#[test]
+fn test_csharp() {
+    let name = simple_build("./examples/csharp-cli");
+    let output = run_image(name);
+    assert!(output.contains("Hello, World!"));
```

Comment:
I think this should be
```suggestion
    assert!(output.contains("Hello world from C#"));
```